### ISSUE TITLE
How to configure Hazelcast cluster address

### ISSUE THREAD

**Issue by attiand** [eclipse/jetty.project#4636]
**Jetty version**

jetty:9.4.26-jre11-slim (Docker image)

**Java version**

jetty@34b2e7432e9f:/var/lib/jetty$ java --version
openjdk 11.0.6 2020-01-14
OpenJDK Runtime Environment 18.9 (build 11.0.6+10)
OpenJDK 64-Bit Server VM 18.9 (build 11.0.6+10, mixed mode)

**Question**
I try to use the session-store-hazelcast-remote module and I seem to have it working with a single hazelcast server on 127.0.0.1. The problem is that I can’t figure out how to configure the url to hazelcast. The [docs](https://www.eclipse.org/jetty/documentation/9.4.27.v20200227/configuring-sessions-hazelcast.html) mentions `jetty.session.hazelcast.configurationLocation` but it seems to have no effect.

I guess for it to work it must be referenced in jetty `hazelcast/src/main/config/etc/sessions/hazelcast/remote.xml`

**Comment by olamy** [comment]
This sounds to be an issue.
In the file hazelcast/src/main/config/etc/sessions/hazelcast/remote.xml can you try adding 
`<Set name="configurationLocation"><Property name="jetty.session.hazelcast.configurationLocation" default="127.0.0.1" /></Set>`
Bug created here #4647

**Comment by attiand** [comment]
Yes, I have already tried that. I got the address issue solved with the XML configuration, however then I think I ran into this issue: https://github.com/eclipse/jetty.project/issues/3571. So I tried to configure the SessionDataSerializer in the XML as it is done in the default configuration `org.eclipse.jetty.hazelcast.session.HazelcastSessionDataStoreFactory`. But then I runned into class loading problems. 

The only way I managed to resolve the problem was to add the address configuration in a modified version of `org.eclipse.jetty.hazelcast.session.HazelcastSessionDataStoreFactor`.
